apps/frontend/src/app/Enrollment/index.tsx

```diff
@@ -228,24 +231,23 @@ export default function Enrollment() {
       .sort((a, b) => a.timeDelta - b.timeDelta); // set doesn't guarantee order, so we sort by timeDelta
   }, [outputs]);
 
-  function updateGraphHover(data: {
-    isTooltipActive?: boolean;
-    chartY?: number;
-    activePayload?: Array<{ value?: number; dataKey?: number }>;
-  }) {
-    if (!data.isTooltipActive || data.chartY === undefined) return;
-    // figure out closest series to mouse that has data point at that value
-    const mousePercent =
-      ((-data.chartY + CHART_HEIGHT) / CHART_HEIGHT) * dataMax;
-    const filteredSeries =
-      data.activePayload?.filter((p) => p.value !== undefined) ?? [];
-    const minDiff = Math.min(
-      ...filteredSeries.map((fs) => Math.abs((fs.value ?? 0) - mousePercent))
-    );
-    const best = filteredSeries.find(
-      (fs) => Math.abs((fs.value ?? 0) - mousePercent) === minDiff
+  function updateGraphHover(data: { activeLabel: string | undefined }) {
```

Comment:
I think instead of defining the type of `data`, it's better to define the function type as `CategoricalChartFunc`:

```ts
const udpateGraphHover: CategoricalChartFunc = (data) => {}
```